src/apps/orders/services.py

```diff
@@ -0,0 +1,127 @@
+from sqlalchemy import delete, insert, select, update
+from sqlalchemy.orm import Session
+
+from src.apps.orders.models import Order, order_product_association_table
+from src.apps.orders.schemas import OrderInputSchema, OrderOutputSchema
+from src.apps.products.models import Product
+from src.apps.user.models import User
+from src.core.exceptions import (AlreadyExists, DoesNotExist, IsOccupied,
+                                 ServiceException)
+from src.core.pagination.models import PageParams
+from src.core.pagination.schemas import PagedResponseSchema
+from src.core.pagination.services import paginate
+from src.core.utils import if_exists
+
+
+def create_order(session: Session, order_input: OrderInputSchema) -> OrderOutputSchema:
+    order_input_data = order_input.dict()
+
+    if order_input_data:
+        user_id = order_input_data.pop("user_id")
+        user = session.scalar(select(User).filter(User.id == user_id).limit(1))
+        
+        if order_input_data.get("product_ids"):
+            product_ids = order_input_data.pop("product_ids")
+            products = session.scalars(select(Product).where(Product.id.in_(product_ids))).all()
+            
+            if not len(set(product_ids)) == len(products):
+                raise ServiceException("Wrong products!")
+
+        order_create_data = dict()
+        order_create_data["user"], order_create_data["products"] = user, products
+        new_order = Order(**order_create_data)
+        session.add(new_order)
+        session.commit()
+
+    return OrderOutputSchema.from_orm(new_order)
+
+
+def get_single_order(session: Session, order_id: int) -> OrderOutputSchema:
+    if not (order_object := if_exists(Order, "id", order_id, session)):
+        raise DoesNotExist(Order.__name__, order_id)
+
+    return OrderOutputSchema.from_orm(order_object)
+
+
+def get_all_orders(
```

Comment:
Musi byc pofiltrowane po aktualnym userze. Przydaloby sie tez opcje sortowania zamowien, filtrowania po statusie czy cos. Wyobraz sobie ze bys z tego potem korzystal. Jakby sie nie dalo zobaczyc zamowien zaplaconych albo w konkretnej dacie czy cos, to by ciezko bylo uzywac tego 😄